apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -90,7 +90,7 @@ type PaperHandlerStatusEvent =
   | { type: 'VOTER_INITIATED_PRINT'; pdfData: Buffer }
   | { type: 'PAPER_IN_OUTPUT' }
   | { type: 'PAPER_IN_INPUT' }
-  | { type: 'VOTER_CONFIRMED_INVALIDATED_BALLOT' }
+  | { type: 'POLLWORKER_CONFIRMED_INVALIDATED_BALLOT' }
```

Comment:
This was a misnomer; the voter is not actually the one who confirms the ballot is has been invalidated